example.config.py

```diff
@@ -1,5 +1,5 @@
 token = "BOT_TOKEN"
-groq_api_key = "GROQ_API_KEY"
+ai_api_key = "GROQ_API_KEY"
```

Comment:
Name this something that makes it apparent that both a groq and openai key can be entered